My Simple App/Application/ViewController.swift

```diff
@@ -10,147 +10,174 @@ import UIKit
 import CoreLocation
 
 class ViewController: UIViewController {
-  @IBOutlet weak var table: UITableView!
-
-  var validAddresses = [Address]()
-  var invalidAddresses = [Address]()
-
-  @IBAction func pressAddAddress(_ sender: Any) {
-    let alertNewAddress = UIAlertController(
-      title: "New Address", message: "enter a new address", preferredStyle: .alert)
-    alertNewAddress.addTextField { textField in
-      textField.textContentType = .addressCityAndState
-      textField.placeholder = "e.g. Santa Rosa, CA"
+    //  @IBOutlet weak var table: UITableView!
+    
+    lazy var table: UITableView = {
+        let table = UITableView(frame: .zero)
```

Comment:
In the storyboard, the table view is a grouped style. Use the following init to achieve this in the UI code:

```suggestion
        let table = UITableView(frame: .zero, style: .grouped)
```